plugins/org.eclipse.sirius.diagram.ui/plugin.properties

```diff
@@ -791,7 +791,8 @@ AirResizableEditPolicy_autoSizeCommandLabel = Auto Size
 AnonymousUserFixedColorName = <anonymous>
 ArrangeBorderNodesAction_actionText = Linked Border Nodes
 ArrangeBorderNodesAction_commandLabel = Arrange Linked Border Nodes
-ArrangeBorderNodesAction_toolTipText = Arrange all the linked border nodes of the diagram.
+ArrangeBorderNodesAction_toolTipDiagramText = Arrange all the linked border nodes of the diagram.
+ArrangeBorderNodesAction_toolTipText = Arrange the linked border nodes of the selected elements.
 ArrangeBorderNodesAction_toolbarActionText = Arrange Linked Border Nodes
```

Comment:
I think we should differanciate the action name for diagram and node because the behavior is not the same
* `Arrange All Linked Border Nodes` for diagram
* `Arrange Linked Border Nodes` for selection

Reply:
Done